composer.json

```diff
@@ -14,7 +14,7 @@
     "minimum-stability": "stable",
     "require": {
         "php": ">=5.4.0",
-        "behat/behat": "~3.0.12"
+        "behat/behat": "~3.0"
```

Comment:
Please note that behat < 3.0.12 has broken arguments resolution system. So instead of  `~3.0` we should probably use `^3.0.12`